corehq/motech/repeaters/models.py

```diff
@@ -361,6 +361,28 @@ def __unicode__(self):
         return "forwarding cases to: %s" % self.url
 
 
+class CreateCaseRepeater(CaseRepeater):
```

Comment:
debated whether to stick these in `zapier` instead of in the `motech.repeaters`. it made slightly more sense to put them here so I could piggyback off the `create_case_repeat_records` signal (and because they felt generic) but I'm happy to move them out if they feel custom.